sdk/communication/Azure.Communication.CallingServer/tests.yml

```diff
@@ -15,9 +15,9 @@ extends:
         SubscriptionConfigurations:
           - $(sub-config-communication-int-test-resources-common)
           - $(sub-config-communication-int-test-resources-net)
-    Clouds: Public,Int
+    Clouds: Public
     EnvVars:
       # SKIP_PHONENUMBER_LIVE_TESTS skips certain phone number tests such as purchase and release
       SKIP_PHONENUMBER_LIVE_TESTS: TRUE
```

Comment:
why do we have SKIP_PHONENUMBER_LIVE_TESTS  in the CallingServer tests.yml? Is it a copy/paste mistake?

Reply:
Removed. Yes, I think so. Also this SKIP_PHONENUMBER_LIVE_TESTS  is appeared in others' sdk test.yml file as well, might need to clean up as well.

Reply:
Thanks for spotting it. Please feel free to clean those too in your future PRs. I am sure the owners of those projects will appreciate it 😊